bin/controller.py

```diff
@@ -119,10 +123,12 @@ def post_new():
                 models.Snippet.get_by_id(parentid)
             except KeyError:
                 raise ValueError("Parent does not exist")
+        if token and len(token) > 22:
+            raise ValueError("Token must not exceed 22 chars")
```

Comment:
Est-ce qu'il y a une limitation technique qui fait qu'on ne doive pas excéder 22 caractères ? Si c'est le cas elle devrait être documenté dans le commit

Reply:
Il n'y a pas de limitation technique, ca éviterai juste de se retrouver avec des tokens qui font je ne sais pas combien de caractères. 22 est le nombre de caractères générés par `token_urlsafe(16)`

Reply:
Alors on peut peut-être ajouter dans le message d'erreur "as of 16 random bytes base64 encoded"